test/integration/dashboard_test.rb

```diff
@@ -1,7 +1,7 @@
 require 'test_helper'
 
 class DashboardTest < ActionDispatch::IntegrationTest
-  def setup
+  before(:all) do
```

Comment:
You're right, I didn't realize `DatabaseCleaner.clean` was triggering `before(:all)` so it's not useful for integration tests.